cmd/zinject/zinject.c

```diff
@@ -602,6 +605,7 @@ register_handler(const char *pool, int flags, zinject_record_t *record,
 				(void) printf(" range: [%llu, %llu)\n",
 				    (u_longlong_t)record->zi_start,
 				    (u_longlong_t)record->zi_end);
+			(void) printf("  dvas: 0x%x\n", record->zi_dvas);
```

Comment:
The `print_data_handler` should also up updated to output the requested DVAs to inject.  I made this change locally.

```diff
--- a/cmd/zinject/zinject.c
+++ b/cmd/zinject/zinject.c
@@ -364,17 +364,19 @@ print_data_handler(int id, const char *pool, zinject_reco>
                return (0);
 
        if (*count == 0) {
-               (void) printf("%3s  %-15s  %-6s  %-6s  %-8s  %3s  %-15s\n",
-                   "ID", "POOL", "OBJSET", "OBJECT", "TYPE", "LVL",  "RANGE");
+               (void) printf("%3s  %-15s  %-6s  %-6s  %-8s  %3s  %-4s  "
+                   "%-15s\n", "ID", "POOL", "OBJSET", "OBJECT", "TYPE",
+                   "LVL", "DVAs", "RANGE");
                (void) printf("---  ---------------  ------  "
-                   "------  --------  ---  ---------------\n");
+                   "------  --------  ---  ----  ---------------\n");
        }
 
        *count += 1;
 
-       (void) printf("%3d  %-15s  %-6llu  %-6llu  %-8s  %3d  ", id, pool,
+       (void) printf(
+           "%3d  %-15s  %-6llu  %-6llu  %-8s  %-3d  0x%02x  ", id, pool,
            (u_longlong_t)record->zi_objset, (u_longlong_t)record->zi_object,
-           type_to_name(record->zi_type), record->zi_level);
+           type_to_name(record->zi_type), record->zi_level, record->zi_dvas);
 
        if (record->zi_start == 0 &&
            record->zi_end == -1ULL)
```